tests/orm/nodes/data/test_remote.py

```diff
@@ -22,11 +24,80 @@ def remote_data(tmp_path, aiida_localhost):
     return node
 
 
-def test_clean(remote_data):
+@pytest.fixture
+def remote_data_ssh(tmp_path, aiida_computer_ssh):
+    """Return a non-empty ``RemoteData`` instance."""
+    # Compared to `aiida_localhost`, `aiida_computer_ssh` doesn't return an actual `Computer`, but just a factory
```

Comment:
:hear_no_evil: `aiida_computer_ssh` should be compared with `aiida_computer_local`, which in that sense they are similar.. the issue is we don't have "`aiida_ssh`" that would return the actual computer instance, lol :hear_no_evil:

Reply:
yews, so maybe please fix this comment, which I found confusing:
```suggestion
    # `aiida_computer_ssh` doesn't return an actual `Computer`, but just a factory
```

Reply:
I now use the `aiida_computer_local` fixture instead of `aiida_localhost`, and actually create the `Computer` from the factory myself (basically the same single line, which is also the only content of the `aiida_localhost` fixture), such that the behavior between the two `remote_data_[local,ssh]` fixtures is exactly the same, and no comment in the code is needed anymore.